src/engraving/rw/read400/tread.cpp

```diff
@@ -1228,7 +1229,20 @@ void TRead::read(KeySig* s, XmlReader& e, ReadContext& ctx)
             s->setShowCourtesy(e.readInt());
         } else if (tag == "showNaturals") {           // obsolete
             e.readInt();
-        } else if (tag == "accidental") {
+        } else if (tag == "accidental") {             // we need to guess proper concert key
+            Key key = Key(e.readInt());
+            Key cKey = key;
+            if (p && !s->concertPitch()) {
+                Interval v = p->instrument(s->tick())->transpose();
+                if (!v.isZero()) {
+                    cKey = transposeKey(key, v);
+                }
+            }
+            sig.setConcertKey(cKey);
+            sig.setKey(key);
+        } else if (tag == "c_accidental") {
```

Comment:
Reading the new properties must be removed from this reader. We should only read them inside read410

Reply:
> Maybe the reason is that we use the 302 reader for unit tests. See `src/engraving/rw/mscloader.cpp`. But I'm not entirely sure if `mscloader` is indeed used for the unit tests.

But it doesnt explain, why if it is in 400 reader, tests pass. Does it?